pymc/backends/arviz.py

```diff
@@ -264,11 +263,18 @@ def _extract_log_likelihood(self, trace):
         if self.model is None:
             return None
 
+        # TODO: The missing portion of partially observed variables no longer appears in the
+        #  logp graph of model.observed_RVs, nor in varg.tag.observations, altough this is
+        #  expected by `self.log_likelihood_vals_point`
+        # TODO: We no longer need one function per observed variable
```

Comment:
should we open issues to keep these todos in mind?

Reply:
Definitely, something like #5247

Reply:
missed that, thanks! I might add a note that it is beginner friendly provided you know Aesara?

Reply:
For the single function you don't need to know Aesara. The only reason I didn't do it in this PR is because I am not sure how to feed the multiple evaluated values to InferenceData xD

Reply:
then it's easy provided you know ArviZ it seems :sweat_smile:. 

If this generates a single function but that function doesn't vectorize over chain and draw dimensions, it would require retructuring all the log likelihood extraction. It is currently structured as 3 nested loops: variable, chain, draw, with the loop over variable being the outermost one. Not sure if this helps.

Reply:
Yeah that's exactly it. Now you can get rid of the outermost loop per variable. And I agree it's mostly a InferenceData question.

It's not a very pressing issue anyway. Most models only have one observation so it won't matter for those.

Reply:
Can we then link to the GH issues here?

Reply:
Added reminder issues #5255 and #5256